### DISCUSSION TITLE
CSS Enhancements TSML-UI

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#837] [Q&A] [answered]
Decided to take the leap and enabled TSML UI on the site. It's got such a prettier front end and I love it that the meetings listed are from whatever day/time it is instead of the earliest meeting of the current day.

I wanted to make men or women after the meeting name stand out more prominently like it did a while back with TSML Legacy. I added the following to my global CSS. Not sure if this was the right place or even the right syntax.

```
/*UI add slash before men or women  */
#tsml-ui .text-muted::before {content: '/ '; 
color: #878787!important;
font-weight: normal;
}

/*UI change color to green for men or women  */
#tsml-ui .text-muted {
    --bs-text-opacity: 1;
    color: #3D8B37!important;
    font-weight: 600;
}
```
But now all "TC" type meetings seem to inherit this by displaying crossed out addresses as green and bold in the meeting detail.

Is there any way to separate these two? Some of our online meetings still intend to have the status be temporarily closed.

And if that is the case, when I make the change from Location/Group to just Location on the meeting list, is there a way I can have those lines crossed out like it did in the legacy version from a while back? This would be the same version where it had a forward slash followed by men or women?

**Comment by joshreisner** [comment, marked as answer]
rather than using `#tsml-ui .text-muted` as your selector, try `#tsml-ui .name small.text-muted` - this should disambiguate it from the inside page.

FYI these custom selector names could change in the future, we might want to someday change it to `.tsml-name` or  `.tsml-meeting-name` for example. (no current plans to do so however)

**Reply by joshreisner** [reply]
it currently isnt possible to add a line-through on the location for TC meetings, because we don't have a way to disambiguate those location columns from the ones that are not TC

**Reply by Grateful-Viv** [reply]
`#tsml-ui .name small.text-muted` only works on the list page, not the detail page.

**Reply by joshreisner** [reply]
isn't that what you wanted?

**Reply by Grateful-Viv** [reply]
I wanted where it currently says either men or women to be preceded by a forward slash in green color both on the meeting list page and the detail page. But I don’t want the detail page’s location and address to be in green if the meeting is temporarily closed.  If it’s not possible, that’s okay. Thanks.

**Reply by joshreisner** [reply]
ah, ok, you can target the inside page separately with `#tsml-ui h1 small.text-muted`

**Reply by Grateful-Viv** [reply]
Thank you. Everything is working as intended.

Maybe when you change custom selector names that could get added to the details on the update so I'll know it needs to be changed.
